caravel/assets/javascripts/global_functions.js

```diff
@@ -0,0 +1,28 @@
+var $ = window.$ = require('jquery');
```

Comment:
Or are you only talking about the line you mentioned? I only have to replace
```
var $ = window.$ = require('jquery');
var jQuery = window.jQuery = $;
```
with 
```
var $ = require('jquery');
```
, just don't touch the window variable, right?